devicetypes/smartthings/zigbee-multi-button.src/zigbee-multi-button.groovy

```diff
@@ -223,3 +252,45 @@ private getModelBindings(model) {
 	}
 	bindings
 }
+
+private Map parseAduroSmartButtonMessage(Map descMap){
+	def buttonState = "pushed"
+	def buttonNumber = 0
+	if (descMap.clusterInt == zigbee.ONOFF_CLUSTER) {
+		if (descMap.command == "01") {
+		    buttonNumber = 1
+		}else if (descMap.command == "00") {
+		    buttonNumber = 4
+		}else {
+		    return [:]
+		}
+	}else if (descMap.clusterInt == zigbee.LEVEL_CONTROL_CLUSTER) {
+		if (descMap.command == "02") {
+		    def data = descMap.data
+		    def d0 = data[0]
+		    if (d0 == "00") {
+			buttonNumber = 2
+		    }
+		    if (d0 == "01") {
+			buttonNumber = 3
+		    }
+		}
+	}else if (descMap.clusterInt == getAduroSmart_SPECIFIC_CLUSTER()) {
```

Comment:
thanks to some nice syntactic sugar in groovy, you can just type `AduroSmart_SPECIFIC_CLUSTER` without the `get` or `()`. It's essentially the Groovy way to do constants, which is why the whole function name (except for `get`) is typically capitalized.

Reply:
see: `getButtonMap()`

Reply:
fixed.